src/app/shared/services/event/event.service.ts

```diff
@@ -18,12 +18,14 @@ export class EventService {
     DB: new Subject<IEvent>(),
     FLOW: new Subject<IEvent>(),
     LOCAL_NOTIFICATION: new Subject<IEvent>(),
```

Comment:
The feedback and template actions system have the usual issue where both want to reference each other (e.g. feedback module triggers actions and actions can trigger feedback module). Recently a nice workaround was found via dynamic imports, although this time round we instead are using the event emitting system to a) see if it works any more nicely and b) provide easier access to the db system which is already hooked up to events